compiler/test/runner.re

```diff
@@ -108,26 +108,32 @@ let read_stream = cstream => {
   Bytes.to_string @@ Bytes.sub(buf, 0, i^);
 };
 
-let run_output = (~code=0, ~heap_size=?, cstate, test_ctxt) => {
+let run_output = (~code=0, ~num_pages=?, cstate, test_ctxt) => {
   let program = extract_wasm(cstate);
   let file = Filename.temp_file("test", ".gr.wasm");
   Emitmod.emit_module(program, file);
 
   let stdlib = Option.get(Grain_utils.Config.stdlib_dir^);
   let testlibs = Sys.getcwd() ++ "/test-libs";
   let result = ref("");
-  let heap_args =
-    switch (heap_size) {
-    | Some(x) => ["--limitMemory", string_of_int(x)]
-    | None => []
+  let env =
+    switch (num_pages) {
+    | Some(x) =>
+      Array.append(Unix.environment()) @@
```

Comment:
Yeah. It already does this by default. If you _don't_ do this when you're trying to add to it, the tests can't even find `grain` 😂

Reply:
Ah!

Reply:
I feel like this could affect reproducibility in the future (even if this has been the behavior before this PR). Maybe we should just find the appropriate `grain` from the `PATH` variable and then build a minimal environment from that?

Reply:
https://github.com/grain-lang/grain/issues/548